### ISSUE TITLE
Accessibility: Orphaned or Missing Form labels 

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#25]
WCAG 2.0 - 1.1.1 Non-text Content Level (A) - violation 'All form inputs have associated text labels.

1.Go to: data.ottawa.ca
2.Observe the below image of identified wave tool 'red' image identifing either;
 A. Missing form label missing - does not have a corresponding form label.
 B. Orphaned form label - a form label is present but not associated with the form.

Resolution: Add text labels to all form inputs. Ensure all orphaned form lables are associated with the form.

![featured datasets_missing form labels](https://cloud.githubusercontent.com/assets/5271253/8011231/fa772042-0b84-11e5-8968-965f2ca26375.JPG)
